src/library/features/get-user-playlists/ui/UserPlaylistsList.tsx

```diff
@@ -0,0 +1,81 @@
+import { TiPin } from "react-icons/ti";
+
+import { Image } from "@/shared/ui/data-display";
+import { Link } from "@/shared/ui/navigation";
+import { GetUserPlaylistsFeature } from "../model/GetUserPlaylistsFeature";
+
+export async function UserPlaylistsList() {
+  const getUserPlaylistsFeature = new GetUserPlaylistsFeature();
+
+  const { data, error } = await getUserPlaylistsFeature.execute();
```

Comment:
The three components are making the exact same request. I think that would mean hat each time a user wants to change the display view, we'll hit the api? This is not ideal, we should try to decorrelate the ui from this.

Something like:
```tsx
function ParentComponent() {
    // Pseudo-code for this line. We may have to turn the component to a client component
    const { data: playlists } =  await getUserPlaylists()

    const [displayMode, setDisplayMode] = useState('list');
 
    return (
        {displayMode === 'list' && <UserPlaylistsList playlists={playlists} />}
        {displayMode === 'grid' && <UserPlaylistsGrid playlists={playlists} />}
        {displayMode === 'compact' && <UserPlaylistsCompact playlists={playlists} />}
    )   
}
```